docs/tools/vega-wallet/concepts.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Wallet Concepts
+hide_title: false
+---
+Vega Wallet is an app that lets you manage wallets and cryptographic key pairs. Key pairs are used to sign and send transactions in a secure way. 
+
+Read on to understand the basic concepts of wallets and keys, and how they work. 
+
+## Cryptographic keys
+Each cryptographic key pair includes a public key and a private key. 
+
+Those keys are associated with an entity that needs to authenticate their identity, or sign and send transaction data. The signature is generated through combining the signer's private key with the data being sent in the transaction. Data that is signed with the private key can only be verified with its corresponding public key.
+
+Signing a transaction proves the signer is the owner of the collateral used in the transaction. The network's validator nodes check and authenticate transactions automatically as they come in.  
```

Comment:
It's more than verifying "collateral". A signature can only be crafted by someone that has the private key. So anything signed by this private key means that it has been "acknowledged" by the signer. It's the ultimate proof to verify the authenticity of something.